lib/parsers/page.js

```diff
@@ -34,7 +34,7 @@ function Page(attrs) {
     }
   }
 
-  this.url = '/' + (this.path.indexOf(path.sep) >= 0 ? path.dirname(this.path) : '')
```

Comment:
I'm not sure if I understand correctly.
If file name is "index.html" then we strip "index.html" from url?
For example:
about/index.html -> /about
about/someotherpage.html -> /about/someotherpage.html